wordpress-mobile/gutenberg-mobile
# Fix missing translation strings

Currently when you run the build script. We end up with deleted translations. 
This is because we accidentally exclude all the *.js files even though we also say that we should include them. This results in the translations that are only defined in the mobile files to not be removed. 

This Pr fixes this by making sure that *.js files are included as expected.

To test:
- Run `npm run genstrings:ios ` - notice that the translations files stay the same.
- RUn `npm run genstrings:android` - notice that the translations files stay the same.

PR submission checklist:
